user-resource/src/main/java/com/andressag/agenda/user/resource/UserResource.java

```diff
@@ -24,25 +24,74 @@
         this.repository = repository;
     }
 
-    @GetMapping
+    @RequestMapping(path = "/users")
```

Comment:
The `@RequestMapping` can remain at class level, specifying the root path (`/users`). For this method, you can continue to use `@GetMapping` as before. No path required - it will default to the path defined at the class level.